table/server/common/src/main/java/alluxio/table/common/udb/PathTranslator.java

```diff
@@ -68,6 +68,10 @@ public PathTranslator addMapping(String alluxioPath, String ufsPath) {
   public String toAlluxioPath(String ufsPath) throws IOException {
     for (BiMap.Entry<String, String> entry : mPathMap.entrySet()) {
       if (ufsPath.startsWith(entry.getValue())) {
+        // return ufsPath if key and value are same.
```

Comment:
nit: Add some description under which case the key and value will be same.

Reply:
Ok